apps/web/lib/settings/personal-setting-form.tsx

```diff
@@ -137,6 +137,8 @@ export const PersonalSettingForm = () => {
 					id: user.id
 				});
 			}
+
+			window.location.replace(`/${newLanguage}/settings/personal`);
```

Comment:
@Cedric921 is this the right way to route to another "page"? I though you need some router or something...

Reply:
I change it to next router